doc/user_guide/user_guide.md

```diff
@@ -71,67 +66,92 @@ The format of the connection object is as following:
-    The CA bundle path should have the following format:
-    ```
-        /buckets/<service-name>/<bucket-name>/<path-to-the-file-or-directory>
-    ```
-    For example, if the service name is ``bfs_service1`` and the bundle was uploaded with the above curl command, the path should look like
-    ``/buckets/bfs_service1/bucket1/tls/ca_bundle.pem``.
-    Please note that for the BucketFS on a SaaS database, the service and bucket names are fixed at
-    respectively ``upload`` and ``default``.
-- service_name: Name of the BucketFS service.
-
-**SaaS Database**
-- url: Optional rrl of the Exasol SaaS. Defaults to 'https://cloud.exasol.com'.
-- account_id: SaaS user account ID.
-- database_id: Database ID.
-- pat: Personal Access Token.
+
+Below is the description of the parameters that need to be passed for On-Prem and SaaS databases.
+
+The distribution of the parameters among those three JSON strings do not matter.
+
+However, we recommend to put secrets like passwords and or access tokens into the `<BUCKETFS_PASSWORD>` part.
+
+#### Parameters of the Adress Part of the Connection Object
+
+`<BUCKET_ADDRESS>` contains various subparameters, e.g.
+
+```sql
+{"url": "https://my_cluster_11:6583", "bucket_name": "default", "service_name": "bfsdefault"}
+```
+
+The number of subparameters and their names depended on the type of database you are connecting to:
+
+**SaaS Database**:
+
+* `url`: Optional URL of the Exasol SaaS. Defaults to https://cloud.exasol.com.
+* `account_id`: SaaS user account ID.
+* `database_id`: Database ID.
+* `pat`: Personal access token.
+
+**On-Prem Database**:
+
+* `url`: URL of the BucketFS service, e.g. `http(s)://127.0.0.1:2580`.
+* `username`: BucketFS username (generally, different from the DB username, e.g. `w` for writing).
+* `password`: BucketFS user password.
+* `bucket_name`: Name of the bucket in the BucketFS, e.g. `default`.
+* `service_name`: Name of the BucketFS service, e.g. `bfsdefault`.
+* `verify`: Optional parameter that can be either a boolean, in which case it controls whether the server's TLS certificate is verified, or a string, in which case it must be a path to a CA bundle to use. Defaults to ``true``.
 
 Here is an example of a connection object for an On-Prem database.
-  ```sql
-  CREATE OR REPLACE CONNECTION "MyBucketFSConnection"
-      TO '{"url":"https://my_cluster_11:6583", "bucket_name":"default", "service_name":"bfsdefault"}'
-      USER '{"username":"wxxxy"}'
-      IDENTIFIED BY '{"password":"wrx1t09x9e"}';
-  ```
+
+```sql
+CREATE OR REPLACE CONNECTION "MyBucketFSConnection"
+    TO '{"url":"https://my_cluster_11:6583", "bucket_name":"default", "service_name":"bfsdefault"}'
+    USER '{"username":"wxxxy"}'
+    IDENTIFIED BY '{"password":"wrx1t09x9e"}';
+```
+
 For more information please check the [Create Connection in Exasol](https://docs.exasol.com/sql/create_connection.htm?Highlight=connection) document.
 
+#### Custom Certificates and Certificate Authorities
```

Comment:
for this section, maybe we should add subsections for "upload bundle to on-prem" and "upload bundle to saas"

Reply:
I agree, see last push for an update.